.pre-commit-config.yaml

```diff
@@ -12,7 +12,7 @@ repos:
   hooks:
   - id: python-check-blanket-noqa
 - repo: https://github.com/timothycrosley/isort
```

Comment:
we have been using `https://github.com/PyCQA/isort`, wondering if there is a difference